extendr-api/src/robj/rinternals.rs

```diff
@@ -357,13 +368,13 @@ pub trait Rinternals: Types + Conversions {
     #[cfg(feature = "non-api")]
     /// Return true if this is a valid string.
     fn is_valid_string(&self) -> bool {
-        unsafe { Rf_isValidString(self.get()).into() }
+        unsafe { extendr_ffi::Rf_isValidString(self.get()).into() }
```

Comment:
Why is this not imported too?

Reply:
Its non-api and behind the non-api feature flag. Feels easier like this as a one liner